packages/editor/src/store/test/selectors.js

```diff
@@ -1097,6 +1098,28 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( 'isPostAutosavingLocked', () => {
+		it( 'should return true if the post has postAutosavingLocks', () => {
+			const state = {
+				postAutosavingLock: { example: true },
+				currentPost: {},
+				saving: {},
+			};
+
+			expect( isPostAutosavingLocked( state ) ).toBe( true );
+		} );
+
+		it( 'should return false if the post has no postAutosavingLocks', () => {
+			const state = {
+				postAutosavingLock: {},
+				currentPost: {},
+				saving: {},
+			};
+
+			expect( isPostAutosavingLocked( state ) ).toBe( false );
+		} );
+	} );
+
```

Comment:
In `isEditedPostAutosaveable` we should add a new test which covers the case when the autosave lock is enabled.

Reply:
@gziolo thanks for the review and feedback.

> In isEditedPostAutosaveable we should add a new test which covers the case when the autosave lock is enabled.


Ok, I'll try to add that.